### PR TITLE
Update react-native docs

### LINKED ISSUE #4705: No file path/name as a comment in most snippets for react ios tutorial
**Environment Information**
Browser: Google Chrome v58.0.3029.96 (64-bit)
URL: https://auth0.com/docs/quickstart/native/react-native

**Description**
Most of the snippets presented in these tutorials don't have the name of the file and the path commented in the first line. The ones that do only have the name of the file in the comment

**Screenshot**
screen shot 2017-07-19 at 10 10 30 pm

### PR DESCRIPTION
**Pull request by paulioceano**
Fix #4705

### CODE REVIEW

File: articles/quickstart/native/react-native/_includes/_login.md [hunk 1]
```diff
@@ -23,6 +25,8 @@ In the file `android/app/src/main/AndroidManifest.xml` you must make sure the **
 So your **MainActivity** should look like this:
 
 ```xml
+//android/app/src/main/java/com/auth0sample/MainActivity.java
```
Comment (cocojoe) [inline articles/quickstart/native/react-native/_includes/_login.md:28]: This is not correct, the `MainActivity` is a sub section inside the `//android/app/src/main/AndroidManifest.xml`

**Review by cocojoe — CHANGES_REQUESTED** [review]
In general please be consistent with the formatting in some cases it is `//comment` others `// comment` I prefer the space.

**Review by aaguiarz — CHANGES_REQUESTED** [review]
@paulioceano check @cocojoe's comments

**Comment by cocojoe** [comment]
@aaguiarz can this be assigned to the 3DI guys to complete?

**Comment by albertoperdomo** [comment]
Requested changes have been addressed in PR #5749. Closing this one.

**Comment by cocojoe** [comment]
ok

**Pull request closed without merging**